sos/plugins/openvswitch.py

```diff
@@ -83,6 +83,29 @@ def setup(self):
                     "ovs-ofctl show %s" % br
                 ])
 
+                # Flow protocols currently supported
+                flow_versions = ["OpenFlow10",
```

Comment:
I'd prefer this in the conventional sos list style:

```python
   foo = [
        "bar",
        "baz",
       "qux"
    ]
```

But I'll fix it up when I merge - this PR has been hanging around long enough already.